packages/binding/src/services/completion/providers/binding.ts

```diff
@@ -17,6 +17,7 @@ import {
 import { BindContext } from "../../../types";
 import { createInitialSnippet } from "./create-initial-snippet";
 import {
+  getAltTypesPrime,
```

Comment:
This function name is very confusing. In general I would avoid using abbreviations especially if matches a commonly used word, because you do not always have the context to figure out what they mean.

Reply:
Done in this patch https://github.com/SAP/ui5-language-assistant/pull/741/commits/17c9fd1c8474f8d2f5f5d7046e37841f27e43a1b